build.gradle

```diff
@@ -49,6 +49,11 @@ if (hasProperty('cfLocal')) {
   def cfHome = String.valueOf(System.getenv('CHECKERFRAMEWORK'))
   dependencies {
     testImplementation files(cfHome + '/checker/dist/checker.jar')
+    testImplementation files(cfHome + '/checker/dist/checker-qual.jar')
+    testImplementation fileTree(dir: cfHome + '/framework-test/build/libs',
+                                include: '*.jar',
+                                exclude: ['*-sources.jar', '*-javadoc.jar'])
+    testImplementation libs.plume.utils
   }
```

Comment:
Can you move the released dependencies to an else block here?
Currently, you first have the released dependencies and then add the local dependencies. In which order are these used?
It seems clearer to split these up cleanly.